packages/k8s-triliovault-operator/k8s-triliovault-operator.patch

```diff
@@ -4,7 +4,7 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-origin
 @@ -10,3 +10,6 @@
  sources:
  - https://github.com/trilioData/k8s-triliovault-operator
- version: v2.0.2
+ version: v2.0.5
```

Comment:
can you patch this version to `2.0.5`? We have an updating upcoming that requires valid semver, and the `v` prefix will cause problems as it is not valid semver.

Reply:
I have fixed this while generating  the chart. I still doubt if this would fix future issues may arise due to changes in your scripts. The prefix 'v' is present in the operator code itself which is already released. I would suggest engg team to remove it altogether from the next operator release.